docs/python_support/python_support.md

```diff
@@ -0,0 +1,748 @@
+# Python servables in OpenVINO Model Server {#ovms_docs_python_support_python_support}
+
+## Introduction
+
+**This feature is in preview, meaning some behviors of the feature as well as user interface are subjects to change in the future versions**
+
+ Starting with version 2023.3, model server supports execution of custom Python code. Users can now create their own servables in Python. Those servables can run simple pre or post processing tasks as well as complex ones like image or text generation. 
```

Comment:
We are not serving Python scripts separately hence they are not servables. They are part of existing servable - mediapipe.
```suggestion
 Starting with version 2023.3, model server supports execution of custom Python code. Users can now create their own servables in Python. Those scripts can run simple pre or post processing tasks as well as complex ones like image or text generation. 
```

Reply:
Makes sense, but in that case there are more places that require changes. I'll remove references to "servables".